nsv13/code/modules/overmap/overmap.dm

```diff
@@ -185,6 +187,8 @@
 	var/flak_battery_amount = 0
 	var/broadside = FALSE //Whether the ship is allowed to have broadside cannons or not
 	var/plasma_caster = FALSE //Wehther the ship is allowed to have plasma gun or not
+	var/crail = FALSE //Do we have a coax railgun? yes or no
```

Comment:
This is not telling the pilot what weapons they have, this is just for telling the code theres a railgun and to add the railgun datum. Pilot uses copied gunner code that has been made pilot specific. I'll add more comments to clear this up :)

Reply:
Ah I forgot about the weapons datum stuff. That's OK then.

Reply:
Actually this is poor coding, and inconsistent with other weapons, as we absolutely should not be holding vars for these.
Weapons are applied on the ship datum rather than generically.

The only reason any var should be added there for specific weapons is if they have special handling.

Reply:
Agree here, railgun datum will be used as long as it's got a slot in apply_weapons and doesn't need special handling for its firing. For example there's no variable for having NAC, BSA, or blue laser here.